packages/sources/src/components/form/models/form-group.base-model.ts

```diff
@@ -10,6 +10,7 @@ export interface FormGroupBase<TemplateFnReturnType> {
   disabled?: boolean;
   infoButton?: InfoButton;
   info?: Info<TemplateFnReturnType>;
+  inline?: boolean;
```

Comment:
Dit model is de basis voor elke form-group. Elke property die hier wordt toegevoegd moet in elke form group worden geimplementeerd.

Reply:
Hmm, ik heb hem nu verplaatst naar waar hij gebruikt wordt, in dit geval de `radios`